cloudify_vsphere/devices/__init__.py

```diff
@@ -226,3 +228,365 @@ def _get_card_ip(client_config, server_id, nic_name):
     server_client.get(config=client_config)
     vm = server_client.get_server_by_id(server_id)
     return server_client.get_server_ip(vm, nic_name, ignore_local=False)
+
+
+def temp_stop_server(cl, server, instance):
+    if server.obj.summary.runtime.powerState.lower() == "poweredoff":
+        return
+    task = server.obj.PowerOff()
+    cl._wait_for_task(task, instance=instance)
+
+
+def temp_start_server(cl, server, instance):
+    if server.obj.summary.runtime.powerState.lower() == "poweredon":
+        return
+    task = server.obj.PowerOn()
+    cl._wait_for_task(task, instance=instance)
+
+
+@operation(resumable=True)
+def create_usb_device(ctx, **kwargs):
+    usb_properties = ctx.instance.runtime_properties
+    usb_properties.update(kwargs)
+    ctx.logger.info("Properties {0}".format(repr(usb_properties)))
+
+
+@operation(resumable=True)
+def delete_usb_device(**kwargs):
```

Comment:
```suggestion
def clean_device_properties(**kwargs):
```
Since this tasks does not actually delete the usb device, when should not describe the task that way.